### ISSUE TITLE
Support numbered lists

### ISSUE THREAD

**Issue by Bilge** [ekalinin/github-markdown-toc#26]

**Comment by csonuryilmaz** [comment]
I agree. I tried with [this file](https://github.com/csonuryilmaz/heybot/blob/master/heybot/README.md) but it didn't worked for me. So I built manually. 

Is there a special reason for not supporting numbered lists?

**Comment by ekalinin** [comment]
No, there's no any special reasons for not supporting numbered lists. Patches are welcome ;)

**Comment by csonuryilmaz** [comment]
Thanks, I don't know whether I will be  successful or not but at the first opportunity I will fork and try to add that feature.

**Comment by csonuryilmaz** [comment]
I forked and worked about 2-3 hours on that but as being a *bash scripting newbie* I couldn't be successful. It outputs this from numbered headers:
![ekran goruntusu - 11-04-2017 - 23 12 15](https://cloud.githubusercontent.com/assets/7355362/24928922/c976ff2e-1f0c-11e7-8f20-21032f3129df.png)
But it should be like this:
![ekran goruntusu - 11-04-2017 - 23 12 47](https://cloud.githubusercontent.com/assets/7355362/24928947/e86922d6-1f0c-11e7-8123-489d512b3f78.png)

It seems easy but it was tough for me. On the other hand I have learnt some cool things from code while I was working on **gh-md-toc** and take notes for my own bash scripts. :blush:

**Comment by Blackjacx** [comment]
Any progress here?
